### PR TITLE
Fabric 5 website and build updates

### PR DESCRIPTION
**Pull request by ecraig12345**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

Various fixes in the Fabric 5 branch prompted by Fabric 7's release and general build tech advancements:

- Add Fabric 7 to v5 website homepage's dropdown

- Add createFlightConfig script and PR+release build yaml files so we can switch the v5 pipelines to be more similar to the newer pipelines

- Prevent running Prettier in v5 files by default to reduce churn (in case anyone is using Prettier+VS Code+format on save)

- Bump allowed Node version to < 11 (to prevent having to switch Node versions to make small changes)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9474)

### CODE REVIEW

File: azure-pipelines.release.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: packages/office-ui-fabric-react/dist
+      artifactName: "fabric"
+      publishLocation: "Container"
+    displayName: "Publish Artifact: Fabric"
+
+  - script: |
+      node ./common/scripts/install-run-rush.js publish -a -p -b 5.0 -t lts -n $(npmToken) --add-commit-details --set-access-level public
```
Comment (kenotron) [inline azure-pipelines.release.yml]: we currently have 6.0 as "lts" as well... @jdhuntington which should it be?

Comment (ecraig12345) [reply azure-pipelines.release.yml]: Oh good catch, I didn't see that flag...

Comment (ecraig12345) [reply azure-pipelines.release.yml]: Probably the real question is, who's using the `lts` tag and why? Guessing it should continue pointing to 5 and **not** 6.

Comment (KevinTCoughlin) [reply azure-pipelines.release.yml]: Likely folks who come across the project via NPM

image

Comment (ecraig12345) [reply azure-pipelines.release.yml]: Removed the lts tag from this release pipeline

File: azure-pipelines.release.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: packages/office-ui-fabric-react/dist
+      artifactName: "fabric"
+      publishLocation: "Container"
+    displayName: "Publish Artifact: Fabric"
+
+  - script: |
+      node node_modules/@microsoft/rush/bin/rush publish -a -p -b 5.0 -n $(npmToken) --add-commit-details --set-access-level public
```
Comment (kenotron) [inline azure-pipelines.release.yml]: I think this needs a tag or else would be published to latest...

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+variables:
+  - group: fabric-variables
+
+pool:
+  vmImage: 'Ubuntu 16.04'
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '8.x'
```
Comment (kenotron) [inline azure-pipelines.yml]: probably at this point we should move to 10 but that can be done later.

Comment (ecraig12345) [reply azure-pipelines.yml]: I was just leaving it consistent with the existing pipeline, which uses Node 8

File: azure-pipelines.release.yml (hunk 2 above)
Comment (ecraig12345) [reply azure-pipelines.release.yml]: Is `legacy-5.0` okay?

**Pull request merged**